infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -286,6 +287,20 @@ private void flushCache(CorfuMsg msg, ChannelHandlerContext ctx, IServerRouter r
         r.sendResponse(ctx, msg, CorfuMsgType.ACK.msg());
     }
 
+    /**
+     * Handles requests for replicating
```

Comment:
We can't use this, the time it takes to transfer a segment is equal to the time it takes to write it to the address space. In other words, its slow. Also, there are many corner cases that are not considered by the replicateRawData implementation, for example the interleaving of trimmed addresses and non-trimmed addresses. 

I re-implemented it here: https://github.com/CorfuDB/CorfuDB/pull/1004/files